docker.md

```diff
@@ -0,0 +1,46 @@
+# Arlocal
+
+Run a local Arweave gateway-like server.
+
+## Getting Started
+### Prerequisites
+
+
+In order to run this container you'll need docker installed.
+
+* [Windows](https://docs.docker.com/windows/started)
+* [OS X](https://docs.docker.com/mac/started/)
+* [Linux](https://docs.docker.com/linux/started/)
+
+### Usage
+
+```shell
+docker run --name arlocal -p 1984:1984 textury/arlocal
+```
+
+## Built With
+
+* Nodejs v16
+
+## Find Us
+
+* [GitHub](https://github.com/textury)
+* [Textury.org](https://textury.org)
+
+## Contributing
+
+Please read [CONTRIBUTING](README.md#contributing) for details on our code of conduct, and the process for submitting pull requests to us.
```

Comment:
Since docker.md will be deployed to dockerhub, the link to readme should be a full path to github. This file wont exist over dockerhub.

Reply:
Oh that's true 🤭